### PR TITLE
fix: outline for blue buttons

### PR DESCRIPTION
**Pull request by Sboonny**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

Ref #2044

Changed the button color, to our primary one and add spacing for outline, as recommended. 

This doesn't add outline to all primary color buttons, the blue ones only.

there are two blue buttons left, they require refactoring which can be left to last.

### CODE REVIEW

**Comment by Sboonny** [comment]
Every change, I have made the past two days will create conflict with this, so we can add conflict tag until everything is merged

**Comment by ojeytonwilliams** [comment]
Changing the outline really should not mean we have to change 25 files.

Chakra lets you define [colour schemes and variants](https://chakra-ui.com/docs/components/button/theming) of every component. So, rather than finding every blue button and altering its outline, why not create a default variant that has the outline you want?  Then you can just use those components and only need to configure them if there's a special case.

**Comment by ojeytonwilliams** [comment]
Sure, I take the point about the number of changes.  It's just that once these components are set up it's really easy to keep things consistent and future PRs really will just change a few lines of code.

File: client/src/modules/dashboard/Chapters/pages/ChaptersPage.tsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ export const ChaptersPage: NextPageWithLayout = () => {
                         chapterId: chapter.id,
                       }),
                   )
-                  .map(({ colorScheme, size, text, href }) => (
+                  .map(({ size, text, href, colorScheme }) => (
```
Comment (ojeytonwilliams) [inline client/src/modules/dashboard/Chapters/pages/ChaptersPage.tsx:93]:
```suggestion
                  .map(({ colorScheme, size, text, href }) => (
```

File: client/src/modules/dashboard/Chapters/pages/ChaptersPage.tsx [hunk 2]
```diff
@@ -156,7 +156,7 @@ export const ChaptersPage: NextPageWithLayout = () => {
                               chapterId: chapter.id,
                             }),
                         )
-                        .map(({ colorScheme, size, href, text }) => (
+                        .map(({ size, text, href, colorScheme }) => (
```
Comment (ojeytonwilliams) [inline client/src/modules/dashboard/Chapters/pages/ChaptersPage.tsx:159]:
```suggestion
                        .map(({ colorScheme, size, href, text }) => (
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Just to cut down on the diff noise:

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**